pysatNASA/instruments/methods/cdaweb.py

```diff
@@ -645,6 +647,11 @@ def _get_file(remote_file, data_path, fname, temp_path=None, zip_method=None):
     if zip_method == 'zip':
         with zipfile.ZipFile(dl_fname, 'r') as open_zip:
             open_zip.extractall(data_path)
+    elif zip_method == 'gz':
```

Comment:
Are the TIDI files zipped? I don't see a 'zip_method' assigned to the instrument.

Reply:
I found the zip method assignment.

Reply:
Yes, tidi files are zipped with zlib module.